library/Kubernetes/Web/NodeDetail.php

```diff
@@ -111,22 +112,30 @@ protected function assemble(): void
             ]),
             new Labels($this->node->label),
             new Annotations($this->node->annotation),
-            new ConditionTable($this->node, (new NodeCondition())->getColumnDefinitions()),
-            new HtmlElement(
+            new ConditionTable($this->node, (new NodeCondition())->getColumnDefinitions())
+        );
+
+        if (Auth::getInstance()->hasPermission(Auth::SHOW_EVENTS)) {
+            $events = Event::on(Database::connection())
```

Comment:
I think this can be harmonized with the other event queries, i.e.
```php
            $events = Event::on(Database::connection())
                ->filter(Filter::equal('referent_uuid', $this->node->uuid));
```